src/wmnds/components/lists/unordered-list/_unordered-list.scss

```diff
@@ -10,19 +10,24 @@
     display: inline;
   }
 
-  li:before {
-    content: "●";
-    padding-right: $size-xsm;
+  li {
+    position: relative;
+
+    &:before {
+      content: "●";
+      position: absolute;
+      left: -1.5rem;
+    }
   }
 
-  ul,
+  ul:not([class*="wmnds-"]),
   ol {
+    margin-top: $size-md;
     list-style: none;
     padding-inline-start: $size-lg;
 
     li:before {
       content: "–";
-      padding-right: $size-xsm;
     }
   }
 }
```

Comment:
Again in this file...

Can you add some padding-inline-start to this like there is on the child ol/ul elements. 

The reason I ask, is that currently the padding on the root ul/ol element comes from the user-agent (browser) styling.
And as you are doing some negative left stuff below, we want to make sure that no matter the browser there is enough space that the negative left can fall into.

An example is, what if a browser treats ul/ol's as having 0 padding-inline-start, then your -1.5rem left will make the bullet icon appear outside of the element/design.

This is the current user-agent styling in chrome as an example:
![image](https://user-images.githubusercontent.com/13015201/94549385-0a629380-024a-11eb-9efa-726e28b83eb3.png)